src/turnkeyml/llm/README.md

```diff
@@ -0,0 +1,128 @@
+# Turnkey-LLM
+
+Welcome to the project page for `turnkey-llm` (aka, "lemonade" the turnkey LLM Aide)!
+Contents:
+
+1. [Getting Started](#getting-started)
+1. [Install Specialized Tools](#install-specialized-tools)
+1. [Code Organization](#code-organization)
+1. [Contributing](#contributing)
+
+# Getting Started
+
+`turnkey-llm` introduces a brand new set of LLM-focused tools. 
+
+## Install
+
+1. Clone: `git clone https://github.com/onnx/turnkeyml.git`
+1. Create and activate a conda environment:
+    1. `conda create -n tk-llm python=3.10`
+    1. `conda activate tk-llm`
+1. `cd turnkeyml` (where `turnkeyml` is the repo root of your TurnkeyML clone)
+1. Install lemonade: `pip install -e .[llm]`
+    - or `pip install -e .[llm-oga-dml]` if you want to use `onnxruntime-genai` (see [OGA](#install-onnxruntime-genai))
+1. `lemonade -h` to explore the LLM tools
+
+## Syntax
+
+The `lemonade` CLI uses the same style of syntax as `turnkey`, but with a new set of LLM-specific tools. You can read abhout that syntax [here](https://github.com/onnx/turnkeyml#how-it-works).
+
+## Chatting
+
+To chat with your LLM try:
+
+`lemonade -i facebook/opt-125m huggingface-load llm-prompt -p "Hello, my thoughts are"`
+
+The LLM will run on CPU with your provided prompt, and the LLM's response to your prompt will be printed to the screen. You can replace the `"Hello, my thoughts are"` with any prompt you like.
+
+You can also replace the `facebook/opt-125m` with any Huggingface checkpoint you like, including LLaMA-2, Phi-2, Qwen, Mamba, etc.
+
+You can also set the `--device` argument in `huggingface-load` to load your LLM on a different device.
+
+Run `lemonade huggingface-load -h` and `lemonade llm-prompt -h` to learn more about those tools.
+
+## Accuracy
+
+To measure the accuracy of an LLM using MMLU, try this:
+
+`lemonade -i facebook/opt-125m huggingface-load mmlu-accuracy --tests management`
```

Comment:
Installed using `pip install -e .[llm-oga-dml]` and can successfully run `lemonade -i facebook/opt-125m huggingface-load llm-prompt -p "Hello, my thoughts are"` but not `lemonade -i facebook/opt-125m huggingface-load mmlu-accuracy --tests management`

```
Error: unrecognized arguments: mmlu-accuracy --tests management. If `mmlu-accuracy --tests management` was intended to invoke a tool, please run `turnkey -h` and check the spelling and availability of that tool.

▄██████████████▄▐█▄▄▄▄█▌
██████▌▄▌▄▐▐▌███▌▀▀██▀▀
████▄█▌▄▌▄▐▐▌▀███▄▄█▌
▄▄▄▄▄██████████████
```

Is mmlu only available when something else is installed? If so, please update the instructions.

Reply:
its accuracy-mmlu 😓

Reply:
```
(tk-llm) PS C:\work\turnkeyml> lemonade -i facebook/opt-125m huggingface-load accuracy-mmlu --tests management

Building "facebook_opt-125m"
    ✓ Loading Huggingface checkpoint   
    ✓ Measuring accuracy with MMLU   

facebook/opt-125m:
        <built-in function input> (executed 1x)
                Parameters:     125,239,296 (477.75 MB)
                Build dir:      C:\Users\jefowers/.cache/lemonade\facebook_opt-125m
                Status:         Successful build!
                                Dtype:  float32 
                                Mmlu Management Accuracy:       19.417 %
```

Reply:
Fixed in https://github.com/onnx/turnkeyml/pull/225/commits/4b2c85e948e603a0f80d0b8deebd584befdc6695